app/db/index.js

```diff
@@ -1,11 +1,8 @@
-const firebase = require('firebase');
+const mongoose = require('mongoose');
 const config = require('../config');
+const { Market } = require('./models');
 
-if (!firebase.apps.length) {
-  firebase.initializeApp(config.firebase);
-}
-
-const db = firebase.database().ref('live_weller').child('markets');
+mongoose.connect(config.db.uri, { useUnifiedTopology: true, useNewUrlParser: true });
```

Comment:
I've looked into it and tried putting them into the uri, but it seems they have to remain in the options section.

Reply:
Sorry should've been clearer. Meant the other way around, where the URI's just the link to the database with nothing in the query string (`https://URI?query=string&...`) so like `retryWrites` would be an option instead of part of the URI

Reply:
ohhhh, I see, I'll give that a go

Reply:
Looking at [here](https://mongoosejs.com/docs/connections.html#options) and [here](https://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html) it doesn't look like I can put those two into the options, they must just be a part of the URI.  I believe when I generated the URI from Atlas it included those in there as well.